src/components/KCollapse/KCollapse.vue

```diff
@@ -1,6 +1,14 @@
 <template>
   <div class="k-collapse w-100">
+    <div v-if="type === 'tags'">
+      <KCollapseItem
+        :no-collapse="noCollapse"
+        :tags="tags"
+        :tags-appearance="tagsAppearance"
+      />
+    </div>
```

Comment:
The `v-if` should be able to go directly on the `<KCollapseItem>` component without the need for the wrapper `div`